org.jellyfin.JellyfinServer.yml

```diff
@@ -21,16 +23,19 @@ cleanup:
   - /share/info
   - /share/man
   - /share/pkgconfig
+  - /share/vpl/examples
 
 finish-args:
   - --device=dri
   - --filesystem=home/Jellyfin Server Media:create
+  - --filesystem=home/.var/app/org.jellyfin.JellyfinServer/media:create
```

Comment:
this is not needed, the app has access to its own ~/.var/app/$FLATPAK_ID directory.